Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ros:dashing
```

Comment:
Consider using https://github.com/Tiryoh/docker-ros2-desktop-vnc as base image. The xserver trick won't work on macos systems, instead the tiryoh version has an integrated vnc server. Or you can also do both